test/skate/0.14.3.js

```diff
@@ -0,0 +1 @@
me");return n&&n.configurable&&Object.defineProperty(r,"name",{configurable:!0,enumerable:!1,value:t.id,writable:!1}),r}function n(e){function t(){var t=r.create(e);return e.prototype.createdCallback.call(t),t}var r=e.type;return t.prototype=e.prototype,t}function o(e,o){var a=void 0,d=void 0,u=t(o);if(u.id=e,u.isNative=u.type===Z["default"]&&(0,J["default"])()&&(0,W["default"])(e),d=(u["extends"]?document.createElement(u["extends"]).constructor:X).prototype,d.isPrototypeOf(u.prototype)||(u.prototype=(0,P["default"])(Object.create(d),u.prototype)),u.prototype.createdCallback=(0,S["default"])(u),u.prototype.attachedCallback=(0,E["default"])(u),u.prototype.detachedCallback=(0,H["default"])(u),u.prototype.attributeChangedCallback=(0,C["default"])(u),u.isNative){var f={prototype:u.prototype};u["extends"]&&(f["extends"]=u["extends"]),a=document.registerElement(e,f)}else a=n(u),Y(),B["default"].register();return a=r(a,u),(0,P["default"])(a,u),q["default"].set(e,a),a}var a={exports:{}},d=a.exports;Object.defineProperty(d,"__esModule",{value:!0});var u=__1675a7174b713323cc232370699a2714,f=e(u),i=__639a0d2e0f8a90cd72e6197bdb481558,l=e(i),c=__ef86f48ff9050407fed1e142d9fe2629,s=e(c),p=__3add36046399fead5a83243849207ed7,b=e(p),_=__db9d338f5bdfd7eb6a3bcf299ff15f91,v=e(_),w=__83ca289f5309abef55c338a9f7a22385,y=e(w),h=__d31d2ff71da7095bf8886568461d8537,x=e(h),g=__662bde51c096e9d79bf327311ea178e0,m=e(g),O=__bbda433df4ec72c4488e3a2f0e6a59a1,M=e(O),j=__d9d26492984e649e5130081ad32bafd6,P=e(j),A=__2b55a083f45c9ef157662a1dc1674218,E=e(A),k=__9f17962f9aa326a94ed3e5d6f6b172e6,C=e(k),N=__fe1aef0db5b664068b470b21f7c754a5,S=e(N),T=__afcda96357b2c6b7e23ccb9ac8b92f43,L=e(T),z=__46b087e8c15b2e0ebc2c4d4cbc36d975,V=e(z),D=__8e93439e8a566d1586c9903a75a6a785,H=e(D),I=__d8200645c4d96aee6940034d9c030d1f,B=e(I),K=__9cff21a9f41cc9ecfe56139e1040c954,q=e(K),F=__c6f5e18624750ce93a74df6369c85ef0,J=e(F),R=__43714db526496b3dd90353996f6dce09,Z=e(R),G=__164e5750c20526cb74a9e443b730eeff,Q=e(G),U=__6e1dfed2b03894ef63a4b65d5038d223,W=e(U),X=window.HTMLElement,Y=(0,L["default"])(function(){(0,Q["default"])(document.documentElement.childNodes,function(e){for(var t=q["default"].find(e),r=t.length,n=0;r>n;n++)t[n].prototype.createdCallback.call(e);for(var n=0;r>n;n++)t[n].prototype.attachedCallback.call(e)})});return o.create=f["default"],o.emit=l["default"],o.fragment=s["default"],o.init=b["default"],o.properties=v["default"],o.ready=y["default"],o.render=x["default"],o.version=m["default"],d["default"]=o,a.exports=d["default"],a.exports}.call(this),("undefined"==typeof window?global:window).__9a43120147a4f5a325d55b702c284d36=function(){function e(e){return e&&e.__esModule?e:{"default":e}}function t(){return window.skate=d,this}var r={exports:{}},n=r.exports;Object.defineProperty(n,"__esModule",{value:!0});var o=__abb93179bdc0236a6e77d3eae07c991c,a=e(o),d=window.skate;return a["default"].noConflict=t,window.skate=a["default"],n["default"]=a["default"],r.exports=n["default"],r.exports}.call(this);
```

Comment:
There is no easy way in npm to have multiple dependencies on the same version of a package, so I chose to check `0.14.3` in.